pkg/cli/cli.go

```diff
@@ -30,22 +34,22 @@ func getFileTypes() []filetype.FileType {
 }
 
 // Initialize the CLI object with default values
-func Init(searchPath string, excludeDirs []string) CLI {
+func Init(searchPath string, excludeDirs []string, excludeTypes []string) CLI {
```

Comment:
Do you think it makes more sense to have the excludeTypes filtering in the `pkg/finder/fsfinder.go` instead of the CLI? That's where the filtering of the excludeDirs is done and it would keep the list of files that gets sent back from the finder to only what will be validated

Reply:
Oh, I probably missed that, yeah. It makes more sense that way.